src/EPR.RegulatorService.Facade.API/Controllers/ApplicationController.cs

```diff
@@ -33,7 +33,8 @@ public async Task<IActionResult> GovNotification([FromBody] GovNotificationReque
             {
                 _logger.LogInformation($"Attempting to send ApprovedPersonAccepted email");
                 string? messageId = _messagingService.ApprovedPersonAccepted(model);
-                _logger.LogInformation($"Email sent: {messageId}");
+                string logData = string.Format("Email sent:{0}", messageId);
+                _logger.LogInformation("{Message}", logData.Replace('\n', '_'));
```

Comment:
I dont think we need to do this replace, in all the other project this worked 
_logger.LogInformation($"Email sent: {0}", messageId);
not going to add review comments for all logger changes as they all have same change

Reply:
Done.